tableauserverclient/server/endpoint/endpoint.py

```diff
@@ -50,6 +50,10 @@ def _make_request(self, method, url, content=None, request_object=None,
         if content is not None:
             parameters['data'] = content
 
+        logger.debug(u'request {}, url: {}'.format(method.__name__, url))
+        if content:
```

Comment:
For publishing requests, this would print out the whole file content. I think we should limit the length, like its done [here for workbook publish](https://github.com/tableau/server-client-python/blob/master/tableauserverclient/server/endpoint/workbooks_endpoint.py#L348) and also remove that duplicate log line

Reply:
Good point, I'll work on adjusting that!